assists/cdo.py

```diff
@@ -29,6 +29,7 @@
 from lopper import Lopper
 import lopper
 from lopper_tree import *
+from xlnx_pm import subsystem
```

Comment:
While there's nothing structurally wrong with this .. subsystems are not only for xilinx_pm (or cdo)

If we are doing a cleanup, we should put these in a xlnx directory, but not specifically xlnx_pm

Reply:
Initially, I thought of going with "xlnx" instead of "xlnx_pm".
But I am thinking if we do "xlnx", people may start adding other non-cdo generation related xilinx plugins in that directory. I intended this folder to only be limited to assists for cdo generation with subsystems and protections.

Let me know what do you think, I am fine either way.

Reply:
We can always have both. xlnx and xlnx_pm, even if there is very little in xlnx initially.  subsystem in particular has the potential to be used for more than just plm/cdo and even non xilinx applications, so having it in xlnx, prepares it for extension / migration in the future.

Reply:
Okay, sounds good. I will use xlnx for now.